CONTRIBUTING.md

```diff
@@ -91,13 +91,12 @@ Here's how to run the test suite:
 
 ## Vendoring
 
-We keep a cached copy of dependencies within the `vendor/` directory,
-managing updates via [Modules](https://golang.org/cmd/go/#hdr-Module_maintenance).
+We managing updates via [Modules](https://golang.org/cmd/go/#hdr-Module_maintenance).
 
-Pull requests should only include `vendor/` updates if they are part of
+Pull requests should only include `go.mod`, `go.sum` updates if they are part of
 the same change, be it a bugfix or a feature addition.
```

Comment:
I can not quite understand the purpose of this description. Maybe it can be simplified. The Go Modules document already contains all information.

Reply:
/cc @lunny @techknowlogick 
```suggestion
```

Reply:
I think the proposed change is correct and should remain (see below for the rationale).

Reply:
The old text was talking about `vendor` directory. But now, that is it talking about?

`if they are part of the same change`: what is the `same change` ...
`be it a bugfix or a feature addition` what does the `bug fix` or `feature addition` refer to ....